dahu/editor/src/main/java/io/dahuapp/editor/kernel/module/ContextManager.java

```diff
@@ -42,7 +42,7 @@ public void fullScreen() {
      * This will be called after a presentation has been opened.
      */
     public void setDisableScreencastMenus(boolean val) {
-        this.primaryStage.getScene().lookup("#"+DahuApp.CAPTURE_MENU_ID).setDisable(val);
```

Comment:
The fix is not integrated in our branch. I need that to execute dahu and forgot it before pushing.